source/Octopus.Tentacle.Tests.Integration/Support/TentacleConfigurationTestCase.cs

```diff
@@ -45,8 +52,8 @@ public override string ToString()
             StringBuilder builder = new();
 
             builder.Append($"{TentacleType},");
-            string version = Version?.ToString() ?? "Latest";
-            builder.Append($"{version}");
+            var version = Version?.ToString() ?? "Latest";
+            builder.Append($"{version},{ScriptServiceToTest.Name.Replace("IAsyncClient", string.Empty)}");
```

Comment:
Just removing the `IAsyncClient` here for brevity (so we just get `ScriptServiceV2` etc in the test name)